src/assets/i18n/fr.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
You can remove the french file, just english is enough, don't want to spend time on adding french later on.

Also we usually use UPPER_SNAKE_CASE for these properties instead of camelCase for distinctive recognition in html files.

Reply:
1. We can choose to keep the french file by doing that efficiently. Reason being, this file it will serve others as an example if they want to use multiple languages. We'll let them remove/add more based on their needs.

Your concern for time spend is right. Regarding that, I'm making sure I don't waste more time and get the whole file quickly from en.json by using ai tools.

2. You are right. I have converted all the translation keys to uppercase.